apps/jump-page/src/layouts/page.astro

```diff
@@ -0,0 +1,33 @@
+---
+interface Props {
+	title: string;
+}
+
+const { title } = Astro.props;
+---
+
+<!DOCTYPE html>
+<html ⚡4email data-css-strict>
```

Comment:
⚡⚡⚡

Is this a typo?

Reply:
This was present in the markup - I believe whoever implemented this originally took a template that was supposed to be an email 🤔

Reply:
Fixed - removed this as it is not needed.